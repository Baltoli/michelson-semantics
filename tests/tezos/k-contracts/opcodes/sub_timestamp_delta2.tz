$contract { parameter (pair timestamp int);
storage timestamp;
code { CAR; DUP; CAR; DIP{CDR}; SUB; NIL operation; PAIR}
 } ;
$param (Pair 100 2000000000000000000) ;
$storage 111 ;
$ExpectedOutput -1999999999999999900 ;

