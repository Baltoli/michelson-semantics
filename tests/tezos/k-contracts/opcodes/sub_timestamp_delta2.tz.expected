<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List #Timestamp ( -1999999999999999900 )
  </stack>
  <paramtype>
    pair .AnnotationList timestamp .AnnotationList int .AnnotationList
  </paramtype>
  <storagetype>
    timestamp .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
